src/mrpro/data/AcqInfo.py

```diff
@@ -236,6 +219,23 @@ def spatialdimension_2d(
             user7=tensor(idx['user'][:, 7]),
         )
 
+        # Calculate orientation as rotation matrix from directional cosines
+        def orientation_from_directional_cosine(
```

Comment:
Not sure if this is correct, please double check

Reply:
Isn't this transposed?

Applying the orientation to a 1,0,0 vector should give the slice_dir.
So slice dir hascto be the first column.

Reply:
should be fixed now

Reply:
I think I added a function to rotation to create it from spatial dimensions...

Reply:
very handy